roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -93,6 +93,16 @@
       --selinux-relabel
       --firstboot-command "growpart /dev/sda {{ _root_part }}"
       --firstboot-command "xfs_growfs /"
+      {% if item.value.ip_address is defined %}
+      --firstboot-command "nmcli con add connection.type 802-3-ethernet
+      connection.id eth0
+      connection.interface-name eth0
+      ipv4.method manual
+      ipv4.addresses {{ item.value.ip_address.address }}
+      ipv4.gateway {{ item.value.ip_address.gw }}"
```

Comment:
as a proof of "using script would be better": the quote here. It should be on the next line I think, in order to get the `ipv4.dns` configuration as well :).
That script would also be calling the `nmcli connection up eth0` ;).